src/observable.js

```diff
@@ -162,10 +163,14 @@ extend(Observable.prototype, {
     }
 
     return this;
-  }
+  },
 
+  toESObservable() {
```

Comment:
I think we should add this two methods to the prototype in `index.js` file, for consistency with how other methods are added.